FCMS/templates/landing.jinja2

```diff
@@ -0,0 +1,45 @@
+{% include "header.jinja2" %}
+<body class="landing">
+  <header class="landing-header">
+    <div class="title">
+      <span class="logo">{% include 'inline_svgs/logo.jinja2' %}</span>
+      Fleet Carrier <br>Management System
+    </div>
+
+    <div class="user">
+      <a class="btn btn-secondary" href="/terms">Terms</a>
+      <a class="btn btn-primary" href="/login">Log in</a>
+      <a class="btn btn-primary" href="/register">Register</a>
+    </div>
+  </header>
+
+  <div class="content">
+    <div class="tracked-carriers statistic">
+      <div class="stat">537</div><!-- TODO get actual number of carriers tracked -->
```

Comment:
Replace number with {{ tracked_carriers }} and codebehind will feed in correct value.